Content.Shared/Chemistry/SharedChemMaster.cs

```diff
@@ -169,7 +169,7 @@ public sealed class ChemMasterBoundUserInterfaceState : BoundUserInterfaceState
 
         public ChemMasterBoundUserInterfaceState(
             ChemMasterMode mode, ContainerInfo? inputContainerInfo, ContainerInfo? outputContainerInfo,
-            IReadOnlyList<ReagentQuantity> bufferReagents, FixedPoint2 bufferCurrentVolume,
+            IReadOnlyList<ReagentQuantity> bufferReagents, FixedPoint2 bufferCurrentVolume, //ss220 tweak sort
```

Comment:
Тут пометка не нужна
```suggestion
            IReadOnlyList<ReagentQuantity> bufferReagents, FixedPoint2 bufferCurrentVolume,
```